src/components/ToDoForm.js

```diff
@@ -25,6 +27,17 @@ const ToDoForm = ({ addTask }) => {
                 onChange={handleChange}
                 placeholder="Enter task..."
             />
+            <button 
+            style ={{
```

Comment:
Please indent style={{}} attribute. Since style is an attribute of <button> it has to be indented when compared to the parent element <button>.